f2/src/forms/ContactInfoForm.js

```diff
@@ -1,162 +1,135 @@
 /** @jsx jsx */
 import React from 'react'
-import addrs from 'email-addresses'
 import PropTypes from 'prop-types'
 import { jsx } from '@emotion/core'
 import { Trans } from '@lingui/macro'
-import { Form } from 'react-final-form'
-import { NextAndCancelButtons } from '../components/next-and-cancel-buttons'
-import { Stack } from '@chakra-ui/core'
 import { useStateValue } from '../utils/state'
-import { ErrorSummary } from '../components/ErrorSummary'
-import { Input } from '../components/input'
-import { Field } from '../components/Field'
-import { clientFieldsAreValid } from '../utils/clientFieldsAreValid'
-import { formatPhoneNumber } from '../utils/formatPhoneNumber'
-import { formDefaults } from './defaultValues'
-import { Flex, Icon } from '@chakra-ui/core'
 import { P } from '../components/paragraph'
```

Comment:
Still using the `<P>` component from Chakra-UI. We should be using the component from f2\src\components\formik\paragraph\index.js.